src/main/java/frc/robot/subsystems/drive/SwerveModuleIO_Real.java

```diff
@@ -94,36 +89,27 @@ public SwerveModuleIO_Real(int drivingCANId, int turningCANId, double chassisAng
             .setPositionPIDWrappingMaxInput(SwerveModuleConstants.kTurningEncoderPositionPIDMaxInput));
 
       // Set the PID gains for the driving motor
-      m_drivingPIDController.setP(SwerveModuleConstants.kDrivingP);
-      m_drivingPIDController.setI(SwerveModuleConstants.kDrivingI);
-      m_drivingPIDController.setD(SwerveModuleConstants.kDrivingD);
-      m_drivingPIDController.setFF(SwerveModuleConstants.kDrivingFF);
-      m_drivingPIDController.setOutputRange(SwerveModuleConstants.kDrivingMinOutput,
-            SwerveModuleConstants.kDrivingMaxOutput);
-
       errors += check(m_drivingPIDController.setP(SwerveModuleConstants.kDrivingP));
+      errors += check(m_drivingPIDController.setI(SwerveModuleConstants.kDrivingI));
+      errors += check(m_drivingPIDController.setD(SwerveModuleConstants.kDrivingD));
       errors += check(m_drivingPIDController.setFF(SwerveModuleConstants.kDrivingFF));
       errors += check(m_drivingPIDController.setOutputRange(SwerveModuleConstants.kDrivingMinOutput,
             SwerveModuleConstants.kDrivingMaxOutput));
 
       // Set the PID gains for the turning motor
-      m_turningPIDController.setP(SwerveModuleConstants.kTurningP);
-      m_turningPIDController.setI(SwerveModuleConstants.kTurningI);
-      m_turningPIDController.setD(SwerveModuleConstants.kTurningD);
-      m_turningPIDController.setFF(SwerveModuleConstants.kTurningFF);
-      m_turningPIDController.setOutputRange(SwerveModuleConstants.kTurningMinOutput,
-            SwerveModuleConstants.kTurningMaxOutput);
-
       errors += check(m_turningPIDController.setP(SwerveModuleConstants.kTurningP));
+      errors += check(m_turningPIDController.setI(SwerveModuleConstants.kTurningI));
+      errors += check(m_turningPIDController.setD(SwerveModuleConstants.kTurningD));
       errors += check(m_turningPIDController.setFF(SwerveModuleConstants.kTurningFF));
       errors += check(m_turningPIDController.setOutputRange(SwerveModuleConstants.kTurningMinOutput,
             SwerveModuleConstants.kTurningMaxOutput));
 
       this.chassisAngularOffset = chassisAngularOffset;
-      m_drivingEncoder.setPosition(0);
 
       errors += check(m_drivingEncoder.setPosition(0));
 
+      System.out.println("turning P: " + m_turningPIDController.getP());
```

Comment:
this doesn't need to be here - we don't need print statements in our constructor (i know i just talked to you about this irl but for consistency's sake this is also here)